create-budget/azuredeploy.json

```diff
@@ -16,17 +16,6 @@
         "description": "The total amount of cost or usage to track with the budget"
       }
     },
-    "budgetCategory": {
```

Comment:
Why are we removing this?

Reply:
To simplify the budget document as a default is already set by the budget service. Let me know if you feel strongly about this, i can add it back.

Reply:
seems like it makes the template more broadly applicable if the resource has the capability